platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/Address.java

```diff
@@ -819,7 +819,7 @@ public boolean equals(Object o) {
         }
 
         Address address = (Address) o;
-        return equalsWithoutStake(address) && stake == address.stake;
+        return equalsWithoutStake(address) && stake == address.stake && ownHost == address.ownHost;
```

Comment:
Is there a reason that `ownHost` is checked, here, rather than being added into `equalsWithoutStake`?

Reply:
I can't recreate the error in development that caused me to move it, so I moved it back.   It will cause problems if someone constructs a new address without checking the network to determine the isOwnHost value or if the node was in the set of local nodes to start.